src/Lib/AutoLoginableInterface.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace AuthActions\Lib;
+
+interface AutoLoginableInterface
```

Comment:
What is this Interface for? It seems to only have the purpose of enabling typehint in `AuthUtilsComponent`, where it could be `EntityInterface` as well, which also would generate less confusion with the variable name `$user` there